argo-workflows/workflow-examples/examples.md

```diff
@@ -0,0 +1,13 @@
+# List all examples
+`ls /root/examples`{{execute}}
+
+# Run an example
```

Comment:
should we add a disclaimer here that some examples might not run properly? either a) because they are broken examples that need updating, or b) because of killercoda?

i'll defer to your judgment.

in either case, we can direct users to open an issue on our course repo i suppose.